warehouse/routes.py

```diff
@@ -276,6 +276,14 @@ def includeme(config):
     # RSS
     config.add_route("rss.updates", "/rss/updates.xml", domain=warehouse)
     config.add_route("rss.packages", "/rss/packages.xml", domain=warehouse)
+    config.add_route(
+        "rss.project.releases",
+        "/rss/{name}/",
```

Comment:
I think I'd prefer `/rss/project/{name}/` here, any reason why you chose not to?

Reply:
Looking at #6938, this should probably end in `something.xml` as well.

Reply:
No particular reason, just picked the shortest possible/available. If you want a `.xml` suffix, then (1) `/rss/project/{name},xml` would be one option. Or if you want to leave space for possible other project related feeds, then (2) `/rss/project/{name}/releases.xml` would work, too. Or (3) `/rss/project/{name}/updates.xml` for consistency with the existing `/rss/updates.xml`.

I think I'll go with (2). Hopefully that's ok, if not, please just name it so we save some change cycles here :)

Reply:
I agree, (2) seems like the best option. Thanks!